rest/utilities_testing.go

```diff
@@ -346,7 +346,7 @@ func (rt *RestTester) WaitForNViewResults(numResultsExpected int, viewUrlPath st
 	}
 
 	description := fmt.Sprintf("Wait for %d view results for query to %v", numResultsExpected, viewUrlPath)
-	sleeper := base.CreateDoublingSleeperFunc(20, 10)
+	sleeper := base.CreateSleeperFunc(20, 100)
```

Comment:
Did you want 200, 100 here as well?

Reply:
Yeah, my bad.  Fixed.  (set to 20s -- what do you think?)